packages/web/src/components/gcds-text/gcds-text.css

```diff
@@ -0,0 +1,248 @@
+@layer reset,
+defaults,
+variants.display,
+variants.limit,
+variants.margin,
+variants.role,
+variants.size,
+variants.style,
+variants.weight;
+
+@layer reset {
+  :host {
+    display: block;
+    color: var(--gcds-text-role-primary);
+
+    .gcds-text {
+      box-sizing: border-box;
+      margin: 0;
+    }
+  }
+}
+
+@layer defaults {
+  :host .gcds-text {
+    font: var(--gcds-text-size-body-desktop);
+
+    @media only screen and (width < 48em) {
+      font: var(--gcds-text-size-body-mobile);
+    }
+  }
+}
+
+@layer variants.display {
```

Comment:
Small question: For these display variants to apply, wouldn't they need to be applied to the host element? Since the host element has `display:block`, changing the display property on the `p` tag wouldn't change the how the text displays.

Reply:
Good catch, I updated it!